Cargo.toml

```diff
@@ -14,15 +14,10 @@ readme = "README.md"
 repository = "https://github.com/rust-embedded/embedded-hal"
 version = "0.2.3"
 
-[dependencies.nb]
-version = "0.1.1"
+[dependencies]
+nb = { version = "0.1.1", features = ["unstable"] }
```

Comment:
`unstable` ? That has a weird sound to it considering that we're aiming to stabilize this.

Reply:
This is because currently the `unproven` feature requires `nb/unstable`. The state of the `unstable` feature should be discussed for `nb`'s 1.0 release.